scripts/clear_sessions.py

```diff
@@ -0,0 +1,51 @@
+import sys
+import time
+import logging
+import datetime
+
+from django.db import transaction
+from django.utils import timezone
+
+from framework.celery_tasks import app as celery_app
+from website.app import setup_django
+setup_django()
+from osf.models import Session
+
+from scripts.utils import add_file_logger
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+
+SESSION_AGE_THRESHOLD = 30
+
+
+def main(dry_run=True):
+    old_sessions = Session.objects.filter(modified__lt=timezone.now() - datetime.timedelta(days=SESSION_AGE_THRESHOLD))
+    initial_count = old_sessions.count()
+
+    if dry_run:
+        logger.warn('Dry run mode, will delete files and then abort the transaction')
+    logger.info('Preparing to Delete {} Sessions older than {} days'.format(initial_count, SESSION_AGE_THRESHOLD))
+
+    with transaction.atomic():
+        start = time.time()
```

Comment:
I understand that thought! I decided to keep it in cause it would have been too many checks for dry run to make it worth it to remove it.... but also open to just removing it entirely, I just think it's interesting information!